crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -519,11 +520,17 @@ impl<'w, 's> Commands<'w, 's> {
         self.queue.push(RemoveResource::<R>::new());
     }
 
+    /// Runs the system corresponding to the given [`SystemId`].
+    ///
+    /// Calls [`World::run_system_by_id`](crate::system::World::run_system_by_id).
+    pub fn run_system_by_id(&mut self, id: SystemId) {
```

Comment:
Can we rename this to `run_system`? To avoid confusion, we could rename `RunSystem::run_system` to `run_system_immediate`.

Reply:
I'd suggest something like `run_system_once` instead of `..._immediate` since neither function has any more or less immediacy.

Reply:
Done!